packages/trpc/server/routers/viewer/teams/inviteMember.handler.ts

```diff
@@ -80,14 +88,15 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
           expires: new Date(new Date().setHours(168)), // +1 week
         },
       });
-      if (ctx?.user?.name && team?.name) {
+      if (team?.name) {
         await sendTeamInviteEmail({
           language: translation,
-          from: ctx.user.name,
+          from: ctx.user.name || `${team.name}'s admin`,
```

Comment:
The removed condition in the if-else was preventing sending emails for orgs where the owner had not set their name. Removed the condition and made sure we use a generic `from` property using the team name